samples/RewriteSample/Startup.cs

```diff
@@ -11,14 +11,14 @@ namespace RewriteSample
 {
     public class Startup
     {
-        public void Configure(IApplicationBuilder app, IHostingEnvironment hostingEnvironment)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
```

Comment:
How about mimicking what we do in Configuration? https://github.com/aspnet/Configuration/blob/dev/src/Microsoft.Extensions.Configuration.FileExtensions/FileConfigurationExtensions.cs#L72